homeassistant/components/fritzbox_callmonitor/sensor.py

```diff
@@ -59,7 +59,8 @@
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up Fritz!Box call monitor sensor platform."""
     name = config.get(CONF_NAME)
-    host = config.get(CONF_HOST)
+    # Try to resolve a hostname; if it is already an IP, it will be returned as-is
+    host = socket.gethostbyname(config.get(CONF_HOST))
```

Comment:
The documentation says that `host:` should be an IP address. 

Most users don't run their own DNS server at home thus you need to handle the traceback if the lookup fails.

Reply:
If someone uses a hostname, they should at least have configured a "hosts" file, but I see your point. Added exception catching, and using DEFAULT_HOST as fallback. Or would you prefer an error and not initializing the component at all?

Should I also do a PR for the documentation?